src/main/java/net/fabricmc/loader/transformer/FabricTransformer.java

```diff
@@ -56,6 +59,11 @@
 		ClassVisitor visitor = classWriter;
 		int visitorCount = 0;
 
+		if (applyAccessWidener) {
+			visitor = new AccessWidenerVisitor(Opcodes.ASM7, visitor, FabricLoader.INSTANCE.getAccessWidener());
```

Comment:
```suggestion
			visitor = new AccessWidenerVisitor(Opcodes.ASM8, visitor, FabricLoader.INSTANCE.getAccessWidener());
```

This will need doing when the ASM8 PR is merged.